style.scss

```diff
@@ -2835,3 +2835,397 @@ body >footer {
     
   }
 }
+
+// =======================================================================================
+// education.html
+
+#education {
+  header {
+    text-align: center;
+    padding: 20px 0;
+    
+    @include stripe {
+      background-color: #0D4379;
+//      background-image: url(/images/getsecurity-banner.svg);
+//      background-repeat: no-repeat;
+//      background-position: bottom right;
+    }
+    
+    h1 {
+      width: 85%;
+      paggin-top: 10px;
+      margin: 0 auto;
+      
+      @media (max-width: 850px) {
+        font-size: 35px;
+      }
+      
+      @media (max-width: 750px) {
+        font-size: 30px;
+        
+        br { display: none; }
+      }
+      
+      @media (max-width: 500px) {
+        font-size: 25px;
+      }
+    }
+      
+    p {
+      padding-bottom: 0px;
+      width: 90%;
+      margin: 20px auto;
+      color: #caeaff;
+      
+      @media (max-width: 700px) {
+        font-size: 18px;
+      }
+      
+      @media (max-width: 500px) {
+        font-size: 16px;
+      }
+    }
+    
+    >a {
+      color: white;
+      text-align: center;
+      display: inline-block;
+      width: 38%;
+      margin: 15px;
+      height: 60px;
+      line-height: 60px;
+      font-size: 26px;
+      border-radius: 4px;
+
+      @media (max-width: 900px) {
+        font-size: 22px;
+        margin: 11px;
+        height: 45px;
+        line-height: 45px;
+        width: 42%;
+      }
+
+      @media (max-width: 800px) {
+        font-size: 17px;
+        margin: 8px;
+        line-height: 40px;
+      }
+
+      @media (max-width: 600px) {
+        font-size: 15px;
+      }
+
+      @media (max-width: 430px) {
+        width: 200px;
+      }
+    }
+
+    >a.get {
+      background-image: linear-gradient(to bottom, #884cea, #6a42b8);
+      box-shadow: 0 3px 3px rgba(41,25,66,0.6);
+      background-color: #613c91;
+
+      &:hover {
+        background-image: linear-gradient(to bottom, #925bef, #794cc2);
+        background-color: #794cc2;
+      }
+    }
+
+    >a.demo {
+      background-image: linear-gradient(to bottom, #f46b8b, #d13f6c);
+      box-shadow: 0 3px 3px rgba(41,25,66,0.6);
+      background-color: #d13f6c;
+
+      &:hover {
+        background-image: linear-gradient(to bottom, #fb839f, #d54f79);
+        background-color: #d54f79;
+      }
+    }
+  }
+  
+  section h3 {
+    @media (max-width: 850px) {
+      font-size: 20px;
+    }
+  }
+  
+  li h3 {
+    font-weight: 400;
+  }
+  
+  p {
+    &.highlight {
+      text-align: center;
+      padding-bottom: 20px;
+    }
+
+    &.copyright {
+      text-align: center;
+      font-size: 14px;
+    }
+  }
+  
+  #why-it {
+    
+    >h3 {
+      max-width: 940px;
+      margin: 1em auto;
+    }
+
+    ul {
+      padding: 0;
+      
+      li {
```

Comment:
Consider adding `vertical-align: top` to this element so that side-by-side item titles line up.